### ISSUE TITLE
Statestore not available in third-party ui nodes

### ISSUE THREAD

**Issue by bartbutenaers** [FlowFuse/node-red-dashboard#1067]
Hi guys,

I want to store my dynamic properties (from msg.ui_update) into the State store. However that fails, because only the Data store seems to be available:

![image](https://github.com/FlowFuse/node-red-dashboard/assets/14224149/aea13a94-bd5e-4f16-b222-4739f34d3ac8)

Had a quick look and I think it should be added [here](https://github.com/bartbutenaers/node-red-dashboard/blob/main/nodes/config/ui_base.js#L739) in the ui_base:
```
        node.stores = {
            data: datastore
        }
```

I will implement this change myself during the weekend and test it.  
But then at least it has been officially registered, in case somebody else meanwhile has the same problem.

Bart

**Comment by bartbutenaers** [comment]
That was indeed what I needed.  Seems to be better now after a quick fix:

![image](https://github.com/FlowFuse/node-red-dashboard/assets/14224149/de7fa4c0-4cb8-4be6-85a1-f2a73d090068)

Will create a pull request later on...

**Comment by joepavitt** [comment]
Closed by #1075
